docs/Install-Guide/compiling-rpms.md

```diff
@@ -5,28 +5,28 @@ Creating rpm's of GlusterFS from git source is fairly easy, once you know the st
 
 RPMS can be compiled on at least the following OS's:
 
--   Red Hat Enterprise Linux 5, 6 (& 7 when available)
--   CentOS 5, 6 (& 7 when available)
--   Fedora 16-20
+- Red Hat Enterprise Linux 5, 6 (& 7 when available)
+- CentOS 5, 6 (& 7 when available)
+- Fedora 16-20
 
 Specific instructions for compiling are below. If you're using:
 
--   Fedora 16-20 - Follow the Fedora steps, then do all of the Common steps.
--   CentOS 5.x - Follow the CentOS 5.x steps, then do all of the Common steps
--   CentOS 6.x - Follow the CentOS 6.x steps, then do all of the Common steps.
--   RHEL 6.x - Follow the RHEL 6.x steps, then do all of the Common steps.
+- Fedora 16-20 - Follow the Fedora steps, then do all of the Common steps.
+- CentOS 5.x - Follow the CentOS 5.x steps, then do all of the Common steps
+- CentOS 6.x - Follow the CentOS 6.x steps, then do all of the Common steps.
+- RHEL 6.x - Follow the RHEL 6.x steps, then do all of the Common steps.
 
-Note - these instructions have been explicitly tested on all of CentOS 5.10, RHEL 6.4, CentOS 6.4+, and Fedora 16-20. Other releases of RHEL/CentOS and Fedora may work too, but haven't been tested. Please update this page appropriately if you do so. :)
+**Note** - these instructions have been explicitly tested on all of CentOS 5.10, RHEL 6.4, CentOS 6.4+, and Fedora 16-20. Other releases of RHEL/CentOS and Fedora may work too, but haven't been tested. Please update this page appropriately if you do so. :)
```

Comment:
we need to update these to the latest versions if possible. Shall we go head with a separate issue for this?

Reply:
Same as above. Content changes in separate PR.